selenium/testcontainers/selenium/__init__.py

```diff
@@ -10,18 +10,23 @@
 #    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #    License for the specific language governing permissions and limitations
 #    under the License.
+import uuid
+from pathlib import Path
+from typing import Optional
 
+import urllib3
 from selenium import webdriver
 from selenium.webdriver.common.options import ArgOptions
 from testcontainers.core.container import DockerContainer
 from testcontainers.core.waiting_utils import wait_container_is_ready
-from typing import Optional
-import urllib3
 
+from .video import SeleniumVideoContainer
+
+EMPTY_PATH = "."
 
 IMAGES = {
-    "firefox": "selenium/standalone-firefox-debug:latest",
-    "chrome": "selenium/standalone-chrome-debug:latest"
+    "firefox": "selenium/standalone-firefox:4.13.0-20231004",
+    "chrome": "selenium/standalone-chrome:4.13.0-20231004"
 }
```

Comment:
probably dont want to change the images, is this necessary? or what is the motivation

Reply:
I updated the image because it's outdated and no longer supported.